backend/requirements.txt

```diff
@@ -17,3 +17,4 @@ mlfoundry==0.10.6
 servicefoundry==0.10.1
 markdownify==0.11.6
 gunicorn==21.2.0
+jupyter
```

Comment:
this should not be added here?

Reply:
we need jupyter notebook to run locally, then should I add as a separate pip install statement in the docs?

Reply:
yes